dashboard/app/views/levels/_standalone_video.html.haml

```diff
@@ -0,0 +1,18 @@
+:javascript
+  function getResult() {
+    return {
+      response: 'ok',
+      result: true
+    };
+  }
+
+.standalone-video
+  - video = @level.specified_autoplay_video
+  - src = "/videos/embed/#{video.youtube_code}?rel=0"
```

Comment:
This seems like something that should be absorbed into the Video model. @wjordan is going to help me investigate whether we can use a single fallback path everywhere.